src/components/para_trials/timerInput.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import styles from "../../styles/Paraflow.module.css";
+
+interface TimerInput {
+  onStartTimer: (inputTimeInStandard) => void;
```

Comment:
I would recommend that this component internally stores and provides durations as milliseconds/seconds rather than a string so callers don't have to worry about conversion

Reply:
In order to store the time as a number, I would have to convert the user input on every change of the input string, instead of once when the "start" button is pressed. I could create a function within the component to convert the string to seconds, before being passed to onStartTimer. Please let me know your thoughts on this.

Reply:
yeah converting before passing to `onStartTimer()` seems good